config/AMBASSADORS_MEMBERS.json

```diff
@@ -1,164 +1,176 @@
-				"date": {
-					"year": 2021,
-					"month": "August"
-				},
-				"link": "https://www.youtube.com/watch?v=rwlHJkgaRdw"
-			},
-			{
-				"type": "presentation",
-				"title": "Event-driven APIs & Schema governance for Apache Kafka, API Specification Conference",
-				"date": {
-					"year": 2021,
-					"month": "August"
-				},
-				"link": "https://www.youtube.com/watch?v=QZ4cJyTw-U0"
-			},
-			{
-				"type": "presentation",
-				"title": "Getting started with Event-Driven APIs, Apidays LIVE Paris",
-				"date": {
-					"year": 2021,
-					"month": "December"
-				},
-				"link": "https://www.youtube.com/watch?v=_kIE-qtpwTA&t=9s"
-			}
-		]
-	},
-	{
-		"name": "Jesse Menning",
-		"github": "jessemenning",
-		"twitter": "JesseMenning",
-		"linkedin": "jesse-menning",
-		"company": "Solace",
-		"contributions": [
-			{
-				"type": "article",
-				"title": "AsyncAPI 2.3 Adds Solace Bindings to Unlock Full Power of PubSub+",
-				"date": {
-					"year": 2022,
-					"month": "January"
-				},
-				"link": "https://solace.com/blog/asyncapi-adds-solace-bindings/"
-			},
-			{
-				"type": "article",
-				"title": "Align Production Reality and Event Documentation with the AsyncAPI Discovery Tool",
-				"date": {
-					"year": 2021,
-					"month": "November"
-				},
-				"link": "https://solace.com/blog/asyncapi-discovery-tool/"
-			},
-			{
-				"type": "article",
-				"title": "AsyncAPI vs. OpenAPI: Answers to Your Burning Questions About the Two Leading API Specs",
-				"date": {
-					"year": 2021,
-					"month": "June"
-				},
-				"link": "https://solace.com/blog/asyncapi-vs-openapi/"
-			},
-			{
-				"type": "article",
-				"title": "AsyncAPI, CloudEvents, OpenTelemetry: Which Event-Driven Specs Should Your DevOps Include?",
-				"date": {
-					"year": 2021,
-					"month": "March"
-				},
-				"link": "https://solace.com/blog/asyncapi-cloudevents-opentelemetry-event-driven-specs-devops/"
-			},
-			{
-				"type": "presentation",
-				"title": "Future of AsyncAPI, Panel discussion, AsyncAPI Conference",
-				"date": {
-					"year": 2021,
-					"month": "November"
-				},
-				"link": "https://www.youtube.com/watch?v=3EeMHhbwyOQ"
-			},
-			{
-				"type": "presentation",
-				"title": "Thinking Out Loud #4 — with Jesse Menning",
-				"date": {
-					"year": 2021,
-					"month": "August"
-				},
-				"link": "https://www.youtube.com/watch?v=eJewFUR-OaM"
-			}
-		]
-	}
+  {
+    "name": "Alejandra Quetzalli",
+    "img": "/img/avatars/canela-ale.webp",
+    "bio": "Hecha en 🇲🇽 • 📑OSS DevDocs & Community @asyncAPIspec• 📚Autora @Apress(2023) • Canela🐕‍🦺 es mi #actuallyautistic #ServiceDog • Rebelde🤟🏽• mis opiniones",
+    "title": "DevRel & DevDocs @AsyncAPI",
+    "github": "alequetzalli",
+    "twitter": "QuetzalliAle",
+    "tags": ["presentation", "workshop", "article", "iniciative_driver"],
+    "linkedin": "alejandra-quetzalli",
+    "company": "Postman",
+    "contributions": [
```

Comment:
the idea of having a record of contributions was to later show them in UI. Can you do it?

I wrote some simple suggestion in https://github.com/asyncapi/website/pull/802#issuecomment-1269478313 in the past. But I bet you can propose something better

Reply:
Yeah, I already added a subpage for each ambassador https://deploy-preview-1123--asyncapi-website.netlify.app/community/ambassador-program/alequetzalli

Are you suggesting something in line with this?

Reply:
oh, ok, subpage is a good appraoch